pipeline/workflow-builder.Dockerfile

```diff
@@ -13,7 +21,7 @@ FROM ${BUILDER_IMAGE:-registry.redhat.io/openshift-serverless-1/logic-swf-builde
 # When using nightly:
 # ARG QUARKUS_EXTENSIONS=org.kie:kogito-addons-quarkus-jobs-knative-eventing:999-SNAPSHOT,org.kie:kie-addons-quarkus-persistence-jdbc:999-SNAPSHOT,io.quarkus:quarkus-jdbc-postgresql:3.8.4,io.quarkus:quarkus-agroal:3.8.4,org.kie:kie-addons-quarkus-monitoring-prometheus:999-SNAPSHOT,org.kie:kie-addons-quarkus-monitoring-sonataflow:999-SNAPSHOT
 # When using prod:
-ARG QUARKUS_EXTENSIONS=io.quarkiverse.openapi.generator:quarkus-openapi-generator:2.9.1-lts,org.kie:kie-addons-quarkus-monitoring-sonataflow:10.0.0,org.kie:kogito-addons-quarkus-jobs-knative-eventing:10.0.0,org.kie:kie-addons-quarkus-persistence-jdbc:10.0.0,io.quarkus:quarkus-jdbc-postgresql:3.8.6,io.quarkus:quarkus-agroal:3.8.6
+ARG QUARKUS_EXTENSIONS=io.quarkiverse.openapi.generator:quarkus-openapi-generator:2.9.1-lts,org.kie:kie-addons-quarkus-monitoring-sonataflow,org.kie:kogito-addons-quarkus-jobs-knative-eventing,org.kie:kie-addons-quarkus-persistence-jdbc,io.quarkus:quarkus-jdbc-postgresql:3.15.4.redhat-00001,io.quarkus:quarkus-agroal:3.15.4.redhat-00001
```

Comment:
is `io.quarkiverse.openapi.generator:quarkus-openapi-generator:2.9.1-lts` still needed in OSL 1.36? Is it not the version used already?

Reply:
No, it wasn't included.
I pulled this in the custom builder image as shown in the referenced link from the PR message.

Now the m2 repo contains:
```
ls -l ~/.m2/repository/io/quarkiverse/openapi/generator/quarkus-openapi-generator
total 8
drwxrwxr-x. 2 kogito root   4096 May 20 17:33 2.9.0-lts
drwxr-xr-x. 2 kogito kogito 4096 Jun  1 12:33 2.9.1-lts
```
and we'll have to specify this explicitly for at least another verion.

Reply:
ah, crap, ok
Thx for checking :)
May you add a comment for this one so we do not forget to remove it (or be cold feet)